### PR TITLE
Case repeater

### PR DESCRIPTION
**Pull request by sravfeyn**
http://manage.dimagi.com/default.asp?170129
@orangejenny cc @snopoke

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/case/models.py [hunk 1]
```diff
@@ -822,10 +822,13 @@ def force_save(self, **params):
             self._doc["_rev"] = conflict._rev
             self.force_save()
 
-    def to_xml(self, version):
+    def to_xml(self, version, include_case_on_closed=False):
         from xml.etree import ElementTree
         if self.closed:
-            elem = get_case_element(self, ('close'), version)
+            if include_case_on_closed:
+                elem = get_case_element(self, ('close', 'create', 'update'), version)
```
Comment (snopoke) [inline corehq/ex-submodules/casexml/apps/case/models.py]: Does the order here matter? Thinking 'close' should be at the end.

Comment (sravfeyn) [inline corehq/ex-submodules/casexml/apps/case/models.py]: Good point, will change it. Order shouldn't matter anyway

**Comment by orangejenny** [comment]
looks okay to me, will leave for @snopoke to merge when satisfied

File: corehq/ex-submodules/casexml/apps/case/models.py (hunk 1 above)
Comment (czue) [inline corehq/ex-submodules/casexml/apps/case/models.py]: seems worth using the same order as the spec (so `create` then `update` then `close`)

**Comment by czue** [comment]
didn't review too closely but looks good to me pending the one change @snopoke mentioned!

**Comment by snopoke** [comment]
:white_check_mark: once tests pass

**Pull request merged**